src/main/resources/com/google/api/codegen/go/doc.snip

```diff
@@ -10,6 +10,15 @@
     @join line : view.packageDoc
         // {@line}
     @end
+    //
+    // Use of Context
+    //
+    // The ctx passed to NewClient is used for authentication requests and gRPC
+    // transport setup, but not for future calls performed using the client.
```

Comment:
Can we skip the "gRPC" and say something like "for the underlying connection, which is shared between requests".